test/TestUtilities/SystemUtil.cpp

```diff
@@ -87,7 +87,8 @@ std::vector<std::byte> loadIntoMemory(const std::filesystem::path& path)
     const auto size = file.tellg();
     file.seekg(0, std::ios::beg);
     std::vector<std::byte>       buffer(static_cast<std::size_t>(size));
-    [[maybe_unused]] const auto& result = file.read(reinterpret_cast<char*>(buffer.data()), size);
+    [[maybe_unused]] const auto& result = file.read(reinterpret_cast<char*>(buffer.data()),
+                                                    static_cast<std::streamsize>(size));
```

Comment:
When compiling on Android there was a new conversion warning due to platform-specific difference in the definition of types like `std::size_t` and `std::streamsize`.